### ISSUE TITLE
Error when outputting pages with a directory on the file name

### ISSUE THREAD

**Issue by Cisneiros** [NakedJSX/nakedjsx.github.io#1]
I'm rendering multiple pages from a single file, outputting different language versions of the same page. For the default language, the output file is `index.html`. For the alternative languages (e.g. `pt`), the output file is `pt/index.html`.

Trying to output a page using 
```js
Page.Render("pt/index.html")
```
fails with
```
$ npx nakedjsx src --out out

# ...

Building ...
node:internal/process/promises:289
            triggerUncaughtException(err, true /* fromPromise */);
            ^

[Error: ENOENT: no such file or directory, open '/<site_root>/out/pt/index.html'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/<site_root>/out/pt/index.html'
}

Node.js v20.5.1
```

If I run the following before building:

```
$ mkdir -p out/pt
```

Then the build succeeds and everything works as expected (including the nested page correctly referencing the assets directory with `../asset`).

Is this an expected use case of the tool? Or is this error by design and I should accomplish that differently?

**Comment by Cisneiros** [comment]
By the way, I just now realised I opened this issue on the docs website's repository instead of the actual tool's repository. Feel free to transfer it there. Sorry about that!

**Comment by dqh-au** [comment]
Thanks for the report, I'll fix it :)

**Comment by dqh-au** [comment]
Sorry for delay - this is fixed in 0.14.2
